metadata-ingestion/src/datahub/configuration/config_loader.py

```diff
@@ -100,33 +101,42 @@ def load_config_file(
     squirrel_original_config: bool = False,
     squirrel_field: str = "__orig_config",
     allow_stdin: bool = False,
-    resolve_env_vars: bool = True,
-    process_directives: bool = True,
+    allow_remote: bool = True,  # TODO: Change the default to False.
+    resolve_env_vars: bool = True,  # TODO: Change the default to False.
```

Comment:
What's blocking us from doing this now? Especially `allow_remote` which is new

Reply:
we need to tweak datahub-actions to explicitly set resolve_env_vars=True before we can change the default there

we could change it for allow_remote right now, but didn't want to have too many unrelated changes in this file right now